.github/workflows/deploy.yml

```diff
@@ -0,0 +1,59 @@
+name: Deploy Styleguide
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: yarn install --production=false
+      - name: Download fonts from S3
+        uses: actions/aws/cli@master
+        with:
+          args: s3 cp s3://eds.ethoslabs.io/fonts.zip ${GITHUB_WORKSPACE}/fonts.zip --profile ci-user
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_CONFIG_FILE: ${GITHUB_WORKSPACE}/.aws/config
+      - name: Unzip fonts
+        run: unzip -o ${GITHUB_WORKSPACE}/fonts.zip -d ${GITHUB_WORKSPACE}/src/
+      - name: Build styleguide
+        run: yarn styleguide:build
+      - name: Archive build result
+        uses: actions/upload-artifact@master
+        with:
+          name: styleguide
+          path: styleguide
+
+  deploy:
+    name: Deploy
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Download build files
+        uses: actions/download-artifact@master
+        with:
+          name: styleguide
+      - name: Sync styleguide to S3
+        uses: actions/aws/cli@master
+        with:
+          args: s3 sync ${GITHUB_WORKSPACE}/styleguide s3://eds.ethoslabs.io/ --delete --exclude "fonts.zip" --profile ci-user
```

Comment:
Does `sync` and `delete` here basically equate to an upsert where we're overwriting with new styleguide correct?

Reply:
yeah `--delete` means delete everything that doesn't match the source, `--exclude` except for "fonts.zip"